spec/fixtures/Podfile_UsesFrameworksHash.yaml

```diff
@@ -0,0 +1,17 @@
+---
+installation_method:
+  name: cocoapods
+sources:
+  - https://cdn.cocoapods.org/
+target_definitions:
+  - abstract: true
+    children:
+      - children:
+          - abstract: false
+            inheritance: search_paths
+            name: SampleAppTests
+        name: SampleApp
+        uses_frameworks:
```

Comment:
likewise this is used to test that we can load `uses_frameworks` hash value from a serialized `YAML` `Podfile`.